storage/pipeline/commit_batch.go

```diff
@@ -233,13 +233,6 @@ func (b *CommitBatcher) maybeStartBatch(notif bool) ([]sealiface.CommitBatchRes,
 
 	individual := (total < cfg.MinCommitBatch) || (total < miner.MinAggregatedSectors) || blackedOut() || !cfg.AggregateCommits
 
-	// TODO: remove after nv25 (FIP 0100)
```

Comment:
@tediou5 there's also line 388 in this file, using `policy.AggregateProveCommitNetworkFee`, that's now a noop that could be removed.